infrastructure/containerized-data-importer/README.md

```diff
@@ -0,0 +1,13 @@
+## Containerized data importer
+Is a persistent storage management add-on for Kubernetes. It's primary goal is to provide a declarative way to build Virtual Machine Disks on PVCs for Kubevirt VMs
```

Comment:
What is this component? A service? An operator? "add-on" is not so clear to me.
build Virtual Machine Disks on PVCs for Kubevirt VMs ==> use PVCs to store Virtual Machine disks to be used by Kubevirt VMs?

Reply:
Cdi is the operator that control the import the vm image in the pvc.
Actually it is not very clear we will change this description

Reply:
the change " use PVCs to store Virtual Machine disks to be used by Kubevirt VMs" looks good to me